core/src/checks/self_checks.c

```diff
@@ -68,6 +68,12 @@ int run_self_checks(void) {
     ERROR("self check failure: verify_conv_btc_to_satoshi failed.");
   }
 
+  t = verify_rpc_oversized_message_rejected();
```

Comment:
We should actually use C99 bools instead of ints, it's much more readable and we only need 2 values anyway. That refactor can be a separate PR though.